styles/portfolio.css

```diff
@@ -0,0 +1,27 @@
+.portfolio-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    padding-left: 0;
+}
+.portfolio-item {
+    font-weight: bold;
+    background-color: #dab785;
+    margin: 25px;
+    padding: 20px;
+    border: solid;
+}
+.portfolio-item:hover {
+    color:#70a288;
+}
+body {
+    background-color: #d5896f;
+    color: #04395e;
+}
+a {
+    color: #04395e;
+    text-decoration: none;
+}
+a:hover {
+    color: #dab785 
```

Comment:
Style: Missing `;`.

Technically `;`s are only separators in CSS so you _can_ leave it off the last one.  It's just good for preventing bugs so you don't need to remember to add it if you add another rule to the block.